tools/export/iar/ewp.tmpl

```diff
@@ -1,974 +1,1043 @@
                 <option>
-                  <name>ExePath</name>
-                  <state>$PROJ_DIR$\BUILD\Exe</state>
+                    <name>CoreVariant</name>
+                    <version>24</version>
+                    <state>{{device.CoreVariant}}</state>
+                </option>
+                <option>
+                    <name>GFPUDeviceSlave</name>
+                    <state></state>
+                </option>
+                <option>
+                    <name>FPU2</name>
+                    <version>0</version>
+                    <state>{{device.FPU2}}</state>
+                </option>
+                <option>
+                    <name>NrRegs</name>
+                    <version>0</version>
+                    <state>{{device.NrRegs}}</state>
+                </option>
+                <option>
+                    <name>NEON</name>
+                    <state>{{device.NEON}}</state>
+                </option>
+                <option>
+                    <name>GFPUCoreSlave2</name>
+                    <version>24</version>
+                    <state>{{device.GFPUCoreSlave2}}</state>
+                </option>
+                <option>
+                    <name>OGCMSISPackSelectDevice</name>
+                </option>
+                <option>
+                    <name>OgLibHeap</name>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>OGLibAdditionalLocale</name>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>OGPrintfVariant</name>
+                    <version>0</version>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>OGPrintfMultibyteSupport</name>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>OGScanfVariant</name>
+                    <version>0</version>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>OGScanfMultibyteSupport</name>
+                    <state>0</state>
+                </option>
+                <option>
+                    <name>GenLocaleTags</name>
+                    <state></state>
+                </option>
+                <option>
+                    <name>GenLocaleDisplayOnly</name>
+                    <state></state>
+                </option>
+                <option>
+                    <name>DSPExtension</name>
+                    <state>{{device.DSPExtension}}</state>
+                </option>
+                <option>
+                    <name>TrustZone</name>
+                    <state>{{device.TrustZone}}</state>
+                </option>
+                <option>
+                    <name>TrustZoneModes</name>
+                    <version>0</version>
+                    <state>{{trustZoneMode}}</state>
```

Comment:
Any problem with making this variable match the name of this option?
```suggestion
                    <state>{{TrustZoneModes}}</state>
```

Reply:
No problem in particular, I guess that choice was made since all other variables in python code had small initial char 
https://github.com/ARMmbed/mbed-os/pull/9808/files/66e93fd51a768cfe97c8d47e67f708f62ed8b178#diff-873b68ad68a5b46172dca8f9f82f5d13R142

Reply:
AH true, I agree. Maybe just the plural then?
```suggestion
                    <state>{{trustZoneModes}}</state>
```

Reply:
Ok yeah lets not change it for an `s`, it's Friday and my brain is gone 😆